src/test/kotlin/com/example/pass_manager/service/PassServiceTest.kt

```diff
@@ -0,0 +1,91 @@
+package com.example.pass_manager.service
+
+import com.example.pass_manager.domain.MongoPass
+import com.example.pass_manager.repositories.PassRepository
+import com.example.pass_manager.service.impl.PassServiceImpl
+import com.example.pass_manager.util.PassFixture.passFromDb
+import com.example.pass_manager.util.PassFixture.passToCreate
+import com.example.pass_manager.util.PassFixture.passes
+import com.example.pass_manager.util.PassOwnerFixture.passOwnerFromDb
+import com.example.pass_manager.util.PassOwnerFixture.passOwnerId
+import com.example.pass_manager.util.PassOwnerFixture.updatedOwner
+import io.mockk.every
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import io.mockk.justRun
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.data.repository.findByIdOrNull
+import java.time.Instant
+
+@ExtendWith(MockKExtension::class)
+internal class PassServiceTest {
```

Comment:
```suggestion
internal class PassServiceImplTest {
```

We test the concrete service here, not interface.

Don't forget about the package